mechanisms/terminator/micm/species.json

```diff
@@ -5,14 +5,49 @@
       "type": "CHEM_SPEC",
       "absolute tolerance": 1e-12,
       "molecular weight [kg mol-1]": 0.035453,
-      "__is advected": true
+      "__is advected": true,
+      "__is parameterized": false
     },
     {
       "name": "Cl2",
       "type": "CHEM_SPEC",
       "absolute tolerance": 1e-12,
       "molecular weight [kg mol-1]": 0.070906,
-      "__is advected": true
+      "__is advected": true,
+      "__is parameterized": false
+    },
+    {
+      "name": "O2",
+      "type": "CHEM_SPEC",
+      "tracer type": "CONSTANT",
+      "molecular weight [kg mol-1]": 0.0319988,
+      "__is advected": false,
+      "__is parameterized": true,
+      "__is added to profile": true,
+      "__unit" : "molecule cm-3",
+      "__scale height [km]": 7.0
+    },
+    {
+      "name": "O3",
+      "type": "CHEM_SPEC",
+      "tracer type": "CONSTANT",
+      "molecular weight [kg mol-1]": 0.0479982,
+      "__is advected": false,
+      "__is parameterized": true,
+      "__is added to profile": true,
+      "__unit" : "molecule cm-3",
+      "__scale height [km]": 7.0
+    },
+    {
+      "name": "air",
+      "type": "CHEM_SPEC",
+      "absolute tolerance": 1e-12,
+      "molecular weight [kg mol-1]": 0.0289644,
+      "__is advected": false,
+      "__is parameterized": false,
+      "__is added to profile": true,
+      "__unit" : "molecule cm-3",
+      "__scale height [km]": 8.01
```

Comment:
Hi @mattldawson, Is it okay to include air in the species.json The goal is to avoid hardcoding the gas species in atmospheric physics configuration

Reply:
I think that's fine, if CAM-SIMA expects air to be in the constituents array. I would just add a `"tracer type": "CONSTANT"` so that the solver doesn't treat it as a variable to be solved for.

Reply:
Okay, thank you! I will fix that. Hi @nusbaume, @peverwhee and @jimmielin, is it acceptable to add air into the constituent array?

Reply:
Hi @boulderdaze, I also believe that should be fine, but it might be good to call it `dry_air` or something like that ( assuming it is not including H2O as part of the mixture, which CAM-SIMA might complain about).  Of course if that doesn't work for you please let me know!

Reply:
Yeah, I'll use `dry_air`. Thank you for your suggestion!